src/guide_restli_server.md

```diff
@@ -846,7 +857,8 @@ Valid types for query parameters are:
 -   Custom types (see the bottom of this section)
 -   Record template types (any subclass of `RecordTemplate` generated
     from a `.pdl` schema)
--   Arrays of one of the types above, e.g. `String[]`, `long[]`, ...
+-   Arrays of one of the types above, e.g. `String[]`, `long[]`, ... (Remember
```

Comment:
1. Did you leave out Maps because those aren't supported as a parameter? Or did you forget?
2. The comment in parentheses seems to imply that creating a typeref is the same as using `String[]`. I would alter this part so that it's phrased contrastingly like `(Though it's recommended to define datamodels for arrays and maps by creating a typeref)` (may be helpful to also mention maps here too, since users may also be curious how to define parameters of a map type)